oioioi/contests/templatetags/get_user_name.py

```diff
@@ -77,3 +86,8 @@ def public_name(parser, token):
 def full_name(parser, token):
     (user, asvar) = _get_name(parser, token, "full_name")
     return FullNameUserInfoLinkNode(user, asvar)
+
+@register.tag
+def name_with_permissions(parser, token):
```

Comment:
How about `name_and_badge` as TemplateTag name?

Reply:
Good idea, thank you!